controllers/servicediscovery/servicediscovery_controller.go

```diff
@@ -351,6 +359,39 @@ func newLighthouseCoreDNSService(cr *submarinerv1alpha1.ServiceDiscovery) *corev
 	}
 }
 
+func updateDNSCustomConfigMap(client controllerClient.Client, k8sclientSet clientset.Interface, cr *submarinerv1alpha1.ServiceDiscovery,
```

Comment:
Couldn’t this share a lot of `updateDNSConfigMap`?

Reply:
Except for section that adds forward to lighthouse, it is different. Note how with way AKS sets it up, we modify just our part of configmap and don't have to worry about retaining rest of existing configmap.

Reply:
OK, let’s get this in, I’ll follow up with a PR to show what I mean.